daemon/c_service.c

```diff
@@ -229,6 +233,9 @@ c_service_cb_accept(int fd, unsigned events, event_io_t *io, void *data)
 
 	IF_FALSE_RETURN(events & EVENT_IO_READ);
 
+	service->clients++;
```

Comment:
should this be moved after the check s.t. service->clients is only increased when a conneciton is actually established?